docs/update_examples.py

```
@@ -197,6 +210,20 @@ def new_examples(config_file, project_accession=None):
         report_path = project.get("path")
         file_type = project.get("file_type")
 
+        # Validate required fields
+        if not accession:
+            print(f"⚠️ Skipping project: missing accession field")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)